ChrisPerez/server-with-persistance/test/test.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const chai = require('chai');
+const fs = require('fs');
+const chaiHTTP = require('chai-http');
+chai.use(chaiHTTP);
+const expect = chai.expect;
+const request = chai.request;
+require(__dirname + '/../server.js');
+
+describe('HTTP tests', ()=>{
```

Comment:
I had to create a /data directory to get the test to pass.  This could go in a before.